app/src/main/java/us/shandian/giga/ui/fragment/MissionsFragment.java

```diff
@@ -224,9 +224,15 @@ private void updateList() {
         mList.setAdapter(mAdapter);
 
         if (mSwitch != null) {
-            mSwitch.setIcon(mLinear
-                    ? ThemeHelper.resolveResourceIdFromAttr(requireContext(), R.attr.ic_grid)
-                    : ThemeHelper.resolveResourceIdFromAttr(requireContext(), R.attr.ic_list));
+            boolean isLight = ThemeHelper.isLightThemeSelected(mContext);
+            int icon;
+
+            if (mLinear)
+                icon = R.drawable.ic_grid_white_24dp;
+            else
+                icon = R.drawable.ic_list_white_24dp;
+
+            mSwitch.setIcon(icon);
```

Comment:
Here, too. Also, `isLight` is unused.
```suggestion
            mSwitch.setIcon(mLinear
                    ? R.drawable.ic_grid_white_24dp
                    : R.drawable.ic_list_white_24dp);
```